installers/osie/main.go

```diff
@@ -119,6 +119,8 @@ func kernelParams(ctx context.Context, action, state string, j job.Job, s ipxe.S
 	switch j.PlanSlug() {
 	case "c2.large.arm", "c2.large.anbox":
```

Comment:
Could we add the recently released "c3.large.arm" here?

Reply:
Added in a following commit